org-formation/710-tgw/_tasks.yaml

```diff
@@ -0,0 +1,166 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'tgw'
+
+  accountId:
+    Type: String
+    Description: The identifier for the transit gateway account
+    Default: !Ref TransitAccount
+
+TransitGateway:
+  Type: update-stacks
+  Template: tgw.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}'
+  StackDescription: The transit gateway
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+  OrganizationBindings:
+    AllBinding:
+      Account:
+        - !Ref accountId
+      IncludeMasterAccount: false
+  Parameters:
+    Principals:
+      - !Ref ITSandboxAccount
```

Comment:
Setting this will share the TGW to the it sandbox account which will setup a route from the TGW to the `dustbunny` VPC it sandbox account.  I added these two acconts to test the service.  Once we merge we can add more account/VPCs as needed.